### PR TITLE
Socket Server

### PR DESCRIPTION
**Pull request by Shylexx**
Web Socket Server and Python Client for Gestures.

Hand Tracker serves socket server.
Python Client receives socket server and makes requests to raspberry pi (currently on LAN)
Requires Flask and Websockets from PIP for python client/API

### CODE REVIEW

File: python/APICalling.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+import requests
+import json
+
+# Input from Java must be named "input"
+
+
+def get_status(colour):
+    parameter_1 = colour
+    sql_string = "http://192.168.43.65:5000/{p1}".format(p1=parameter_1)
```
Comment (AlmasB) [inline python/APICalling.py:11]: Why is this named `sql_string`?

File: python/APICalling.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    return requests.get(sql_string).content
+
+
+def set_status(colour):
+    parameter_1 = colour
+    sql_string = "http://192.168.43.65:5000/colour/{p1}".format(p1=parameter_1)
```
Comment (AlmasB) [inline python/APICalling.py:17]: same comment as above

File: python/APICalling.py (hunk 1 above)
Comment (AlmasB) [inline python/APICalling.py:11]: Can we extract the hardcoded ip and port into modifiable variables at the top of the source file

**Review by AlmasB — COMMENTED** [review]
This looks really good, thanks. Please see comments attached.

File: python/APICalling.py (hunk 1 above)
Comment (Shylexx) [reply python/APICalling.py]: This is done, now using the args the script is run with to get IP and port.
Just asking Emil about the sql_string and will change as soon as I know

**Comment by Shylexx** [comment]
I've changed the name over and extracted the IP and port out of the api calling definitions entirely. It uses the CLI args for the IP and the port so it should be easy to setup with ssh as well this way.

**Pull request merged**